src/payment/strategies/masterpass/format-locale.ts

```diff
@@ -0,0 +1,29 @@
+export default function formatLocale(localeLanguage: string): string {
+    const supportedLocales: {[language: string]: string[]} = {es: ['es_es', 'es_mx', 'es_pe', 'es_co', 'es_ar', 'es_cl'],
```

Comment:
Maybe put it into a json file so that it's easier to read considering there is so many countries supported here